Cargo.toml

```diff
@@ -2,13 +2,16 @@
 name = "state-reconstruct"
 version = "0.1.0"
 edition = "2021"
-
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 clap = { version = "4.4.0", features = ["string"] }
-ethers = "2.0.10"
+ethers = "1"
```

Comment:
Why downgrade?

Reply:
The zkSync Merkle Tree library depends on an incompatible (with latest `ethers`) version of `sha2`, which is why I downgraded. However, I think we should monitor this and see if we can upgrade in the future